velox/exec/AggregateInfo.cpp

```diff
@@ -112,13 +112,17 @@ std::vector<AggregateInfo> toAggregateInfo(
       info.function->setLambdaExpressions(lambdas, expressionEvaluator);
     }
 
-    // Sorting keys and orders.
-    const auto numSortingKeys = aggregate.sortingKeys.size();
-    VELOX_CHECK_EQ(numSortingKeys, aggregate.sortingOrders.size());
-    info.sortingOrders = aggregate.sortingOrders;
-    info.sortingKeys.reserve(numSortingKeys);
-    for (const auto& key : aggregate.sortingKeys) {
-      info.sortingKeys.push_back(exprToChannel(key.get(), inputType));
+    auto* entry = getAggregateFunctionEntry(aggregate.call->name());
```

Comment:
nit: perhaps, add a short comment to explain this logic

```
// Ignore sorting properties if aggregate function is not sensitive to the order of inputs.
```

It would be nice to also mention this in the function comments in velox/exec/AggregateInfo.h

Reply:
Done.